examples/seismic/abc_methods/README.md

```diff
@@ -0,0 +1,25 @@
+#  Absorbing boundary conditions for the acoustic wave equation  - Applications in seismic problems.
+
+## Authors: Felipe A. G. Silva, Saulo R. M. Barros and Pedro S. Peixoto
+   (felipe.augusto.guedes@gmail.com, saulo@ime.usp.br, pedrosp@ime.usp.br)
+
+**Important Informations:** These notebooks are part of the Project Software Technologies for Modeling and Inversion (STMI)  at RCGI in the  University of Sao Paulo. The authors are from Institute of Mathematics and Statistics - Applied Mathematics Department.
+
+The objective of these notebooks is to present several schemes which are designed to reduce artificial reflections on boundaries in the numerical solution of the acoustic wave equation with finite differences. We consider several methods, covering absorbing boundary conditions and absorbing boundary layers. Among the schemes, we have implemented:
+
+- Clayton's A1 and A2 boundary conditions;
```

Comment:
No need for semicolons I guess

Reply:
Just to organize the text and help uer to identify the methods that will b used.